README.md

```diff
@@ -100,7 +100,7 @@ Apart from coding, I also maintain a blog - you can find my articles on my [blog
 <br/>
 <hr/>
 
-Connect with moi:
+Connect with me:
```

Comment:
This is intentionally set as mòi as that is _me_ in French

Reply:
Thanks for the heads-up.
By the way, I sent you a message on Linkedln and never got your feedback.
My Linkedln name is Paul Munyao